src/components/ha-clickable-list-item.ts

```diff
@@ -60,15 +60,18 @@ export class HaClickableListItem extends ListItemBase {
           padding-right: var(--mdc-list-side-padding, 20px);
           overflow: hidden;
         }
-        :host-context([style*="direction: rtl;"])
-          span.material-icons:first-of-type {
-          margin-left: var(--mdc-list-item-graphic-margin, 16px) !important;
-          margin-right: 0px !important;
+        span.material-icons:first-of-type {
+          margin-inline-start: 0px !important;
```

Comment:
Do we need important?

Reply:
Yes because otherwise margin-right overwrites it with 16 from 
```
:host([graphic=avatar]) .mdc-deprecated-list-item__graphic, :host([graphic=medium]) .mdc-deprecated-list-item__graphic, :host([graphic=large]) .mdc-deprecated-list-item__graphic, :host([graphic=control]) .mdc-deprecated-list-item__graphic {
    margin-left: 0;
    margin-right: var(--mdc-list-item-graphic-margin, 16px);
}
```

Reply:
You should try to use the exact same selector used as you want to override, then you dont need important.